Coffee_loop/src/CoffeeLoop.java

```diff
@@ -0,0 +1,185 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CoffeeLoop {
+    public static void main(String[] args) throws IOException {
+        int countOfWater = 400;
+        int countOfMilk = 540;
+        int countOfCoffeeBeans = 120;
+        int countOfCups = 9;
+        int countOfMoney = 550;
+
+        int[] array = new int[5];
+        array[0] = countOfWater;
+        array[1] = countOfMilk;
+        array[2] = countOfCoffeeBeans;
+        array[3] = countOfCups;
+        array[4] = countOfMoney;
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        while(true) {
+            System.out.println("Write action (buy, fill, take, remain, exit):");
+            System.out.print(">");
+            String action = reader.readLine();
+
+            if (action.equals("buy")) {
+                array = buy(array);
+            }
+
+            if (action.equals("fill")) {
+
+                array = fill(array);
+            }
+            if (action.equals("take")) {
+                array = take(array);
+            }
+            if (action.equals("remain")) {
+                array = remain(array);
+            }
+            if (action.equals("exit")) {
+                exit();
+                break;
+            }
+
+        }
+
+        reader.close();
+    }
+    public static int[]  buy(int[] array1) throws IOException{
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino, back - to main menu:");
+        System.out.print(">");
+        String number = reader.readLine();
+        //reader.close();
+        if(number.equals("1")){
+
+            if(array1[0] < 250)
```

Comment:
please try to use some field which name is show meaning of this if statement, for example why 250 ?

Reply:
done